unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -386,20 +449,49 @@ public async UniTask<List<FriendRequest>> GetFriendRequestsAsync(int sentLimit,
         public void GetFriendsWithDirectMessages(string userNameOrId, int limit) =>
             apiBridge.GetFriendsWithDirectMessages(userNameOrId, limit, 0);
 
-        public FriendRequest GetAllocatedFriendRequest(string friendRequestId) =>
-            friendRequests.ContainsKey(friendRequestId) ? friendRequests[friendRequestId] : null;
+        public FriendRequest GetAllocatedFriendRequest(string friendRequestId)
+        {
+            if (useSocialApiBridge)
+            {
+                return incomingFriendRequestsById.ContainsKey(friendRequestId) ? incomingFriendRequestsById[friendRequestId] :
```

Comment:
You can simplify all `ContainsKey...[]` with `TryGetValue`

Reply:
This PR is blocking the development of the ECS7, the refactor of this method will be done in a future PR.

The new implementation will be:
```
public bool TryGetAllocatedFriendRequest(string friendRequestId, out FriendRequest result) =>
            useSocialApiBridge
                ? incomingFriendRequestsById.TryGetValue(friendRequestId, out result) || outgoingFriendRequestsById.TryGetValue(friendRequestId, out result)
                : friendRequests.TryGetValue(friendRequestId, out result);
                ```
                
But there are lots of usages that need to be updated that are OOS of this PR but will be done